language/ir-testsuite/tests/function_defs/256_locals.mvir

```diff
@@ -0,0 +1,262 @@
+        let x136 : u8;
+        let x137 : u8;
+        let x138 : u8;
+        let x139 : u8;
+        let x140 : u8;
+        let x141 : u8;
+        let x142 : u8;
+        let x143 : u8;
+        let x144 : u8;
+        let x145 : u8;
+        let x146 : u8;
+        let x147 : u8;
+        let x148 : u8;
+        let x149 : u8;
+        let x150 : u8;
+        let x151 : u8;
+        let x152 : u8;
+        let x153 : u8;
+        let x154 : u8;
+        let x155 : u8;
+        let x156 : u8;
+        let x157 : u8;
+        let x158 : u8;
+        let x159 : u8;
+        let x160 : u8;
+        let x161 : u8;
+        let x162 : u8;
+        let x163 : u8;
+        let x164 : u8;
+        let x165 : u8;
+        let x166 : u8;
+        let x167 : u8;
+        let x168 : u8;
+        let x169 : u8;
+        let x170 : u8;
+        let x171 : u8;
+        let x172 : u8;
+        let x173 : u8;
+        let x174 : u8;
+        let x175 : u8;
+        let x176 : u8;
+        let x177 : u8;
+        let x178 : u8;
+        let x179 : u8;
+        let x180 : u8;
+        let x181 : u8;
+        let x182 : u8;
+        let x183 : u8;
+        let x184 : u8;
+        let x185 : u8;
+        let x186 : u8;
+        let x187 : u8;
+        let x188 : u8;
+        let x189 : u8;
+        let x190 : u8;
+        let x191 : u8;
+        let x192 : u8;
+        let x193 : u8;
+        let x194 : u8;
+        let x195 : u8;
+        let x196 : u8;
+        let x197 : u8;
+        let x198 : u8;
+        let x199 : u8;
+        let x200 : u8;
+        let x201 : u8;
+        let x202 : u8;
+        let x203 : u8;
+        let x204 : u8;
+        let x205 : u8;
+        let x206 : u8;
+        let x207 : u8;
+        let x208 : u8;
+        let x209 : u8;
+        let x210 : u8;
+        let x211 : u8;
+        let x212 : u8;
+        let x213 : u8;
+        let x214 : u8;
+        let x215 : u8;
+        let x216 : u8;
+        let x217 : u8;
+        let x218 : u8;
+        let x219 : u8;
+        let x220 : u8;
+        let x221 : u8;
+        let x222 : u8;
+        let x223 : u8;
+        let x224 : u8;
+        let x225 : u8;
+        let x226 : u8;
+        let x227 : u8;
+        let x228 : u8;
+        let x229 : u8;
+        let x230 : u8;
+        let x231 : u8;
+        let x232 : u8;
+        let x233 : u8;
+        let x234 : u8;
+        let x235 : u8;
+        let x236 : u8;
+        let x237 : u8;
+        let x238 : u8;
+        let x239 : u8;
+        let x240 : u8;
+        let x241 : u8;
+        let x242 : u8;
+        let x243 : u8;
+        let x244 : u8;
+        let x245 : u8;
+        let x246 : u8;
+        let x247 : u8;
+        let x248 : u8;
+        let x249 : u8;
+        let x250 : u8;
+        let x251 : u8;
+        let x252 : u8;
+        let x253 : u8;
+        let x254 : u8;
+        let x255 : u8;
+        let x256 : u8;
+        return;
+    }
+}
+// check: "arguments/locals size (256) cannot exceed 255"
```

Comment:
This size seems a little low (especially given the state of the language frontend). Any chance of bumping this to `u16` bound?

Reply:
@dariorussi @vgao1996 

I'm pretty worried about only 256 locals. The compiler is pretty bad at optimizing for this right now. Any chance of increasing this?

Reply:
Yeah we can certainly make it u16. (Will put up a separate PR if we wish.)

Reply:
Fully agree. In the stackless bytecode we are using in the prover, we have removed this already because our transformations introduce even more locals.

Reply:
Cool I'll put up a separate PR to change it to u16 then.